tardis/rest/app/security.py

```diff
@@ -1,103 +1,108 @@
+import secrets
 from ...configuration.configuration import Configuration
 from ...exceptions.tardisexceptions import TardisError
 
 from bcrypt import checkpw, gensalt, hashpw
-from fastapi import Depends, HTTPException, status
-from fastapi.security import OAuth2PasswordBearer, SecurityScopes
-from jose import JWTError, jwt
-from pydantic import BaseModel, ValidationError
+from fastapi import HTTPException, status
 
-from datetime import datetime, timedelta
+# from fastapi.security import SecurityScopes
+# from jose import JWTError, jwt
+from pydantic import BaseModel  # , Field, ValidationError
```

Comment:
Please remove dead code entirely instead of commenting it out.

Reply:
Ok. I think I commented it out because I wanted to reuse it later. I will delete the rest of dead code in other files too.